### DISCUSSION TITLE
Donations to Opensource projects

### DISCUSSION THREAD

**Discussion by julietkilo** [streetcomplete/StreetComplete#3308] [General]
https://linux30.b1-systems.de/
Please vote for Streetcomplete as I did.

**Comment by riQQ** [comment]
At the moment, they are only collecting proposals. As you most likely already proposed StreetComplete, there's no gain in proposing it again. Voting for the proposed projects starts on October 5th.

**Comment by FloEdelmann** [comment]
Voting did start today, but StreetComplete is not on the list. There is "Fossgis e.V. für OpenStreetMap Deutschland" though, so maybe they'll decide to fund some work in StreetComplete if they win a part of the money.

There are many other good projects in the list, too, though, so check them all out!
